bids-validator/bids_validator/rules/top_level_rules.json

```
@@ -217,9 +217,23 @@
       ]
     }
   },
-  "multi_dir_fieldmap": {
+  "fmap_epi_top": {
     "regexp": "^[\\/\\\\](?:acq-[a-zA-Z0-9]+_)?(?:dir-[a-zA-Z0-9]+_)epi\\.json$"
```

```suggestion
    "regexp": "^[\\/\\\\](?:ses-[a-zA-Z0-9]+_)?(?:acq-[a-zA-Z0-9]+_)?(?:ce-[a-zA-Z0-9]+_)?(?:dir-[a-zA-Z0-9]+_)(?:run-[0-9]+_)?(?:chunk-[0-9]+_)?epi\\.json$"
```

I'm going to ignore m0scan for now, since it's in another regex rule.

```yaml
pepolar:
  suffixes:
    - epi
    - m0scan
  extensions:
    - .nii.gz
    - .nii
    - .json
  datatypes:
    - fmap
  entities:
    subject: required
    session: optional
    acquisition: optional
    ceagent: optional
    direction: required
    run: optional
    chunk: optional
```